src/Microsoft.Framework.PackageManager/Utils/LockFileUtils.cs

```diff
@@ -24,34 +22,38 @@ public static LockFileLibrary CreateLockFileLibrary(IPackagePathResolver resolve
             // it has the correct casing that runtime needs during dependency resolution.
             lockFileLib.Name = correctedPackageName ?? package.Id;
             lockFileLib.Version = package.Version;
+            lockFileLib.Sha512 = File.ReadAllText(resolver.GetHashPath(package.Id, package.Version));
 
-            using (var nupkgStream = package.GetStream())
+            // If the shas are equal then do nothing
+            if (previousLibrary?.Sha512 == lockFileLib.Sha512)
             {
-                lockFileLib.Sha512 = Convert.ToBase64String(sha512.ComputeHash(nupkgStream));
+                lockFileLib.Files = previousLibrary.Files;
+                lockFileLib.IsServiceable = previousLibrary.IsServiceable;
             }
-
-            lockFileLib.Files = package.GetFiles().Select(p => p.Path).ToList();
-
-            var installPath = resolver.GetInstallPath(package.Id, package.Version);
-            foreach (var filePath in lockFileLib.Files)
+            else
             {
-                if (!string.Equals(Path.GetExtension(filePath), ".dll"))
+                lockFileLib.Files = package.GetFiles().Select(p => p.Path).ToList();
+                var installPath = resolver.GetInstallPath(package.Id, package.Version);
+                foreach (var filePath in lockFileLib.Files)
                 {
-                    continue;
-                }
+                    if (!string.Equals(Path.GetExtension(filePath), ".dll"))
```

Comment:
Thats fine, we absolutely don't care about those. Find me a nuget package that has that :stuck_out_tongue_closed_eyes: